code/modules/mob/living/living_defense.dm

```diff
@@ -279,3 +281,6 @@
 		. = shield_check.Invoke(attack_type, ., damage_type, silent)
 		if(!.)
 			break
+
+/mob/living/proc/get_fire_resist()
+	return clamp((100 - get_soft_armor("burn", BODY_ZONE_CHEST)) * 0.01, 0, 1)
```

Comment:
The chest body specifier is not needed. It's not used in the generic mob living soft armor function.

Reply:
humans do not override get_fire_resist but do overide get_soft_armor. Therefore, it is still needed or it will runtime.

Reply:
Oh right. I changed that in my own armor refactoring. Woops.